README.md

```diff
@@ -128,4 +154,12 @@ ros2 launch roscon2023_demo ROSCon2023Demo.launch.py
 In a few seconds, the robots should spawn and start moving.  
 For a more in-depth explanation see the [ros2_ws/README.md](ros2_ws/README.md).
```

Comment:
Optionally, you can refer to other documentation (if there is any) or simply note here, that the number of robots can be manually modified.

Reply:
Here, it may be useful to add, that there are different configuration for different levels, that should be setup manually (at least now).